index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">  
+    <title>Personal Portfolio</title>
+</head>
+<body>
+    <div class="wrapper">
+        <header class="menu">
+            <div class="menu-logo">
+                <h1>Welcome</h1>
+            </div>
+            <div class="menu-button">
+                <ul>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                    <li class="bar"> </li>
+                </ul>
+            </div>
```

Comment:
- [ ] Nice again to have The hamburger button on the right but it has to be a nav tag not div
     * The hamburger button is on the right, and has a nav tag